Azure/azure-cli-extensions
# [Stream Analytics] Update Stream Analytics CLI

> Closed https://github.com/Azure/azure-cli/issues/20053

**Description**

Update Stream Analytics CLI.

**Testing Guide**

```
$ azdev test stream-analytics
```
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines


For new extensions:



### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.
